portfolio/src/main/webapp/projects_experience.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jess DeJong - STEP Intern</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="header">
+      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
+      <h1>Jess's Portfolio</h1>
+      <strong>
```

Comment:
I'm going to hold approval until the other PR is merged, just for clarity here.

Reply:
Realized that this header is for projects_experience.html, and not index.html. Had to manually update this header to be correct.